app/components/site-type-select.hbs

```diff
@@ -0,0 +1,12 @@
+<PowerSelect
+  @allowClear={{true}}
+  @loadingMessage="Aan het laden..."
+  @noMatchesMessage="Geen resultaten"
+  @searchMessage="Typ om te zoeken"
```

Comment:
The search arguments don't really do anything at the moment. `@searchEnabled={{true}}` activates it (but the `searchField` needs to be configured as well).

Reply:
No need the search now, I'll delete the message.